client/src/sagas/AuthSaga.js

```diff
@@ -32,16 +32,49 @@ export function * login (api, action) {
   try {
     const { username, password } = action;
     const res = yield call(api.login, {
-      username: username,
+      id: username,
```

Comment:
We will later need to update the auth API endpoint to accept the variable name `username` instead of `id`. This will help keep the variable name consistent within the UI components.